auth-foundation/src/main/java/com/okta/authfoundation/client/internal/NetworkUtils.kt

```diff
@@ -123,7 +123,8 @@ internal suspend fun <T> OidcConfiguration.internalPerformRequest(
 }
 
 private suspend fun OidcConfiguration.executeRequest(
-    request: Request
+    request: Request,
+    returnVerbatimResponse: (Response) -> Boolean = { false },
```

Comment:
It feels like `returnVerbatimResponse` doesn't accurately describe what's happening, or that it's explicitly used when a 429 is encountered. I'd suggest calling this igoreRateLimit or shouldRetry, something like that.

Reply:
I've renamed this to `ignoreRateLimit`, which describes what I'm trying to do a lot better